src/common/mod.rs

```diff
@@ -216,7 +216,11 @@ impl Ord for Label {
     fn cmp(&self, other: &Self) -> Ordering {
         match (self, other) {
             (Label::Int(i1), Label::Int(i2)) => match (i1.signum(), i2.signum()) {
-                (-1, -1) => (-i1).cmp(&(-i2)),
+                (-1, -1) => match i1.cmp(i2) {
+                    Ordering::Less => Ordering::Greater,
+                    Ordering::Equal => Ordering::Equal,
```

Comment:
Is this not equivalent to `i2.cmp(i1)` ?

Reply:
Good point, done.